### ISSUE TITLE
About emptyLineBefore

### ISSUE THREAD

**Issue by islinxuan** [kutsan/stylelint-config-clean-order#8]
```json
"emptyLineBefore": "threshold",
"emptyLineMinimumPropertyThreshold": 5    // larger than 4
```

👆 it is better?  for example: 

- styling with `<col>` effectively,  like this:

```html
<colgroup>
  <col style="background-color: yellow; width: 20%" span="2" />
</colgroup>
```

Only border, background, width and visibility are applied to `<col>` elements. https://www.w3.org/TR/CSS21/tables.html#columns

- emptyLineBefor is unnecessary, like this:

```html
p {
  color: #000;
  background-color: #fff;
}
```

**Comment by kutsan** [comment]
That's a really good idea. But for some reason this config is not working as intended for me. Even with `threshold`, it behaves like `"emptyLineBefore": "always",`. Maybe it's related to https://github.com/hudochenkov/stylelint-order/issues/101?

**Comment by wimhendrikx** [comment]
You should set `emptyLineMinimumPropertyThreshold` in the general options object of `order/properties-order` like you have done with `emptyLineBeforeUnspecified `. That will work.

**Comment by kutsan** [comment]
@wimhendrikx You're right, I should have digged the docs a little more. 😬 

I added this in https://github.com/kutsan/stylelint-config-clean-order/commit/83fbd4039c1d3bb2ba12f0bee4705aba84c76dac and published with version `v2.0.0`. It would be great if you can test it.

**Comment by wimhendrikx** [comment]
It works! Thanks!
